### ISSUE TITLE
Readme instructions explicitly refer to Melody Kramer

### ISSUE THREAD

**Issue by bennovack** [18F/dolores-landingham-bot#203]
Generally not best practice for docs to refer to specific people vs a role/title anyway, this being a fine example of why :-)

**Comment by jessieay** [comment]
good idea, @bennovack ! We don't currently have a role or title for someone who is in charge of onboarding. In fact, I have no idea how that person gets assigned. cc @jeremiak  -- any insights on how we can generally point someone to "the person in charge on onboarding" ? Maybe an email alias?

**Comment by jeremiak** [comment]
Sure, we could set up an email alias (not 100% sure how to do that) or we could just ask them to come to #wg-onboarding in Slack (if they have access to the 18F slack)?

**Comment by jessieay** [comment]
email aliases are gone via gmail -- if we can find a gmail admin, we can do that. Happy to add Slack channel instead, great idea.
